granulate_utils/containers/cri.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import json
+from typing import List, Optional
+
+import grpc  # type: ignore # no types-grpc sadly
+
+import granulate_utils.generated.containers.cri.api_pb2 as api_pb2  # type: ignore
+from granulate_utils.containers.container import Container
+from granulate_utils.exceptions import CriNotAvailableError
+from granulate_utils.generated.containers.cri.api_pb2_grpc import RuntimeServiceStub  # type: ignore
+from granulate_utils.linux.ns import resolve_host_root_links
+
+RUNTIMES = (
+    ("containerd", "/run/containerd/containerd.sock"),
+    ("crio", "/var/run/crio/crio.sock"),
```

Comment:
I think u need to remove `/var` as u then call `resolve_host_root_links`

Reply:
IDK that's the path I found online and it worked for me on minikube (see https://github.com/Granulate/gprofiler/pull/296#issue-1138070577).

Searched online again - found https://github.com/kubernetes/kubernetes/pull/57054/files#diff-29d60439c44e184a292232ca2a7cdc521aabd650e20726af4cdab64fb8ea03ddR35 which shows it as this path